dds_cli/__main__.py

```diff
@@ -91,7 +91,7 @@
     highlight=False,
 )
 
-if sys.argv[1] != "motd":
+if "motd" not in sys.argv:
```

Comment:
The problem is that if "motd" is anywhere in the commands, such as input somewhere, then the motd won't be displayed. 
Example: 
```
(dds-dev-venv) ina@inasPC:~/repos/DDS/dds_cli$ dds project create -t motd
     ︵ 
 ︵ (  )   ︵ 
(  ) ) (  (  )   SciLifeLab Data Delivery System 
 ︶  (  ) ) (    https://delivery.scilifelab.se/ 
      ︶ (  )    Version 1.0.73 
          ︶
Current user: ina-superadmin
Usage: dds project create [OPTIONS]
Try 'dds project create --help' for help.
╭─ Error ──────────────────────────────────────────────────────────────────────────╮
│ Missing option '--description' / '-d'.                                           │
╰──────────────────────────────────────────────────────────
```
vs 
```
(dds-dev-venv) ina@inasPC:~/repos/DDS/dds_cli$ dds project create -t test
     ︵ 
 ︵ (  )   ︵ 
(  ) ) (  (  )   SciLifeLab Data Delivery System 
 ︶  (  ) ) (    https://delivery.scilifelab.se/ 
      ︶ (  )    Version 1.0.73 
          ︶
Important information:
2022-08-08 10:51 - There will be cluster maintenance on August 16th, 9-11 AM. This 
may affect the DDS and cause some disruptions. 

Current user: ina-superadmin
Usage: dds project create [OPTIONS]
Try 'dds project create --help' for help.
╭─ Error ──────────────────────────────────────────────────────────────────────────╮
│ Missing option '--description' / '-d'.                                           │
╰──────────────────────────────────────────────────────────────────────────────────╯
```

So my suggestion is: 
```suggestion
if len(sys.argv) > 1 and sys.argv[1] != "motd":
```
or something similar

Reply:
Mainly a personal preference, imo it doesn't completely make sense. For example. 

* `dds motd ls` will show the motds in a table, then we don't need to show the motds in a list before that
* `dds motd add` will be only by super admins and then we don't need to see all the previous ones when we run that, we might as well just list them before hand, plus the motds are mainly for the users and not for us
* `dds motd deactivate` will probably be preceeded with `dds motd ls` anyway since we need to see which ID to deactivate, so then listing the motds while running is unnecessary, plus it's for the users again and not for us